src/core_read.cpp

```diff
@@ -46,8 +46,24 @@ CScript ParseScript(const std::string& s)
         }
     }
 
+    std::vector<std::string> blocks;
     std::vector<std::string> words;
-    boost::algorithm::split(words, s, boost::algorithm::is_any_of(" \t\n"), boost::algorithm::token_compress_on);
```

Comment:
These changes are overlapping with the work in #13751, here: https://github.com/bitcoin/bitcoin/pull/13751/files#diff-846395c972e400d3933d195c29740098R50. 
It might be worth helping review those changes, and/or considering how to do this without introducing new Boost usage, as the project is trying to reduce it where possible, see https://github.com/bitcoin/bitcoin/projects/3.

Reply:
Thanks for pointing that out. Some of the ways to resolve this:
- function `Split` implemented in #13751 can be used in place of `boost::algorithm::split` introduced here as well, so it may be resolved by either amending to this commit, or to the one #13751, depending on what's first to get to `master`;
- since this change is not that important for this PR, so I can just delete the last commit and it push it when boost's `split` is eliminated in one way or another;
- implement small `split` utility and use it here, but that again overlaps with #13751.

Reply:
See 86315aa for `std::regex`-based tokenizer. Noticed there's no `std::regex` usage throughout code base, so let me know if there's any policy against that.

Reply:
```

std::vector<std::string> Split(std::string const& s, char delim)
{
    std::vector<std::string> result;
    std::istringstream iss(s);

    for(std::string token; std::getline(iss, token, delim); )
    {
        result.push_back(std::move(token));
    }

    return result;
}

```

I am using this simple sample generic code in my work for Split. Maybe it can help to keep it simple.